docs/performance/getting-started.md

```diff
@@ -0,0 +1,73 @@
+# Getting started with Performance Monitoring
+
+##  Basic usage
+
+**TBD** basic explainer
```

Comment:
Idea for what to put in the TBD: "Understand your Angular app's real-world performance with Firebase Performance Monitoring. [Common metrics](https://firebase.google.com/docs/perf-mon/automatic-web) are collected automatically just by injecting `AngularFirePerformance` into your component's constructor (TODO James: is this true?). You can also add custom traces to monitor specific user flows in your app."